Assets/Puppet/PuppetAppCenter.cs

```diff
@@ -6,11 +6,28 @@
 using Microsoft.AppCenter.Unity;
 using UnityEngine;
 using UnityEngine.UI;
+using Microsoft.AppCenter.Unity.Crashes;
+using AOT;
 
 public class PuppetAppCenter : MonoBehaviour
 {
     public Toggle Enabled;
     public Dropdown LogLevel;
+    public PuppetConfirmationDialog userConfirmationDialog;
+
+    static PuppetAppCenter instance;
+
+    private void Awake()
+    {
+        Crashes.SetUserConfirmationHandler(UserConfirmationHandler);
+        instance = this;
+    }
+
+    [MonoPInvokeCallback(typeof(Crashes.UserConfirmationHandler))]
```

Comment:
Is this work on UWP?

@evgeny-pol friendly reminder about CI

Reply:
Not sure, a have a Mac, UWP is not supported